### ISSUE TITLE
Sprite radians and degress

### ISSUE THREAD

**Issue by Chroma72** [pixijs/pixi.js#6660]
In regards to sprite orientation; why does setting rotation = 0 point right, and setting angle = 0 point down?

**Comment by bigtimebuddy** [comment]
Cannot reproduce. These are the same thing:
https://jsfiddle.net/bigtimebuddy/xg3vL1dk/

Please provide a reproduction.

**Comment by Chroma72** [comment]
Hi. So the link you posted is because rotation is in radians, so setting rotation to 0 directly will point straight down. If you had a function that converted DegToRad and did a rotation = DegToRad(0), you would see that it will point to the right. I know that's not your post but thought I'd throw my two cents in.

I figured out my problem and it is pretty obscure. So I had a function called Angle that would get the angle between two points. I renamed the function to AngleDeg and the issue went away. Apparently there's a javascript or PixiJs global function called Angle that was skewing my angle in degrees by 90. Spent about two hours figuring this out. /sigh
